data/json/mutations.json

```diff
@@ -2479,7 +2479,7 @@
         "id" : "PRED1",
         "name" : "Culler",
         "points" : 2,
-        "description" : "You've had a revelation: by killing the weaker creatures, who would only die anyway, you preserve resources for those better able to survive.",
+        "description" : "By killing the weaker creatures, who would only die anyway, you preserve resources for those better able to survive.  You are less bothered by death of others: their weakness invited these fates upon them.",
```

Comment:
The "You've had a revelation:" phrase served an important function here of making it clear that this is a belief that the survivor has, not fact.